pages/pages.go

```diff
@@ -207,65 +210,114 @@ func toHTML(isReception bool, filename string) (string, error) {
 }
 
 // parseDir creates a response for a directory.
-func parseDir(isReception bool, root, dir string) (*Resp, error) {
+func parseDir(isReception bool, root, dir string) (*RespStore, error) {
 	log.WithField("dir", dir).Debug("Parsing directory:")
 
-	// Our content files.
-	var (
-		bodyPath    = filepath.Join(dir, "body.md")
-		sidebarPath = filepath.Join(dir, "sidebar.md")
-		metaPath    = filepath.Join(dir, "meta.toml")
+	const (
+		bodyPattern     = "body(_\\w+)\\.md"
+		sidebarPattern  = "sidebar(_\\w+)\\.md"
+		titlePattern    = "Title(_\\w+)"
+		metaFile        = "meta.toml"
+		iso8601DateTime = "2006-01-02T15:04:05Z"
 	)
 
-	// Parse markdown to HTML.
-	body, err := toHTML(isReception, bodyPath)
-	if err != nil {
-		return nil, err
-	}
-	log.WithField("body", body).Debug("HTML of body.md")
+	bodyReg := regexp.MustCompile(bodyPattern)
```

Comment:
These should probably be made global variables (and the string literals defining them might as well be passed into `MustCompile` directly)